packages/core/src/components/animation-controller/animator.tsx

```diff
@@ -1023,12 +1024,6 @@ export class Animator {
       children[i].progressStep(stepValue);
     }
 
-    if (this._isReverse) {
-      // if the animation is going in reverse then
-      // flip the step value: 0 becomes 1, 1 becomes 0
-      stepValue = ((stepValue * -1) + 1);
-    }
-
```

Comment:
`this._progress(stepValue);` already does reverse the value. Here we were reversing the stepValue twice, making no effect.

Reply:
we probably want to move the comment, it is nice